README.md

```diff
@@ -41,6 +41,10 @@ Options is a map with the following entries:
     :password        The password to bind with, optional
     :num-connections The number of connections in the pool, defaults to 1
     :ssl?            Boolean, connect over SSL (ldaps), defaults to false
+    :cipher-suites   An optional set of strings corresponding to SSL
+                     cipher suites, defaults to ....
+    :ssl-protocols   An optional set of strings corresponding to SSL
+                     protocols, defaults to ....
```

Comment:
thoughts on defaults?

Reply:
I think they should both  default to `nil` for backwards compatibility.

Reply:
For the protocols, it would be good to list the ones we currently support mappings for.